tests/testthat/test-qenv_eval_code.R

```diff
@@ -76,7 +76,7 @@ testthat::test_that("eval_code works with quoted code block", {
 })
 
 testthat::test_that("eval_code fails with unquoted expression", {
-  testthat::expect_error(eval_code(qenv(), a <- b), "object 'b' not found")
```

Comment:
I just checked locally, error message from evaluating this code is unchanged.

Reply:
Sorry, something must have been wrong with the setup on my side. I don't see it anymore either. Reverted the removal of error messages in those tests https://github.com/insightsengineering/teal.code/pull/214/commits/8340d0c4c827f509c3bd600f8051ec0a21b6d739